x-pack/solutions/observability/plugins/apm/server/lib/helpers/get_apm_alerts_client.ts

```diff
@@ -24,6 +24,7 @@ export type ApmAlertsRequiredParams = ESSearchRequest & {
   query?: estypes.QueryDslQueryContainer;
   sort?: estypes.SortOptions[];
   _source?: string[] | false;
+  search_after?: Array<string | number>;
```

Comment:
why is this needed?

Reply:
IIRC, because this way we reduce the actual definition of `search_after` (which includes `boolean` and `null` 🤷 )